docs/pre-release/releaseNotes.yml

```diff
@@ -30,6 +30,35 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.3.4
+    date: '2021-07-09'
+    notes:
+      - type: bugfix
+        title: Improved IP log statements
+        body: >-
+          Fixed log statements that were printing incorrect IP addresses.
+      - type: bugfix
+        title: Improved messaging when multiple services match a workload
+        body: >-
+          If multiple services matched a workload when performing an intercept, Telepresence would panic.
+          It now gives the correct error message, instructing the user on how to specify which
+          service the intercept should use.
+      - type: bugfix
+        title: Traffic-manger determine's service subnet in own namespace
+        body: >-
+          Telepresence will now determine the service subnet by creating a dummy-service in its own
```

Comment:
Did we have existing documentation about how Telepresence determines the service subnet? If so, we should update it; if not, we should add it. In both cases we should link to it here.

Reply:
Will do, the docs were correct, the behavior was not.